glmc4-common/src/main/java/p0nki/glmc4/block/Chunk.java

```diff
@@ -25,7 +25,9 @@ public static Chunk generate(int cx, int cz) {
                 Biome b = Biomes.BIOMES.get(biomes[x][z]);
                 int genX = x + (16 * cx);
                 int genZ = z + (16 * cz);
-                int y = (int) (8 + 4 * generator.generate(genX, genZ));
+                // AAIAIAJUAIAIAIIAIAIAIA
+                int noise = generator.generate(genX, genZ);
+                int y = (int) (8 + 4 * noise + (noise * 0.5f) + (noise * 0.25f));
```

Comment:
Same issue:
```java
int y = (int) (8 + 4 * noise + (noise * 0.5f) + (noise * 0.25f));
```
is equivalent to
```java
int y= (int) (8 + 4.75f * noise);
```
Although I appreciate that you no longer call simplex noise three times more than absolutely necessary, is changing the constant from 4 to 4.75 really going to affect terrain generation that much?
If you really think that increasing the constant by 0.75 is worth doing, please do it by changing 4 to 4.75 instead of adding on in this way, its kind of an unintuitive way to accomplish this.

Reply:
Guys. It's not about increasing the constant. It's about adding the layers. Look up octave. Oise

Reply:
Noise

Reply:
Read the formula.
`int y = (int) (8 + 4 * noise + (noise * 0.5f) + (noise * 0.25f));`
is *exactly* the same as
`int y = (int) (8 + 4.75f * noise);`
There is literally no difference. If it isn't about increasing the constant then this code is incorrect.